### ISSUE TITLE
Gene panels missing?

### ISSUE THREAD

**Issue by 1ctw** [Clinical-Genomics/scout#3638]
Only the gene panels that have been ordered are visible in the list on the first page and in the filter on the variant page. It is not possible to choose to analyze another panel.

**Comment by 1ctw** [comment]
For example case 22331

**Comment by dnil** [comment]
Hi! That is the way we have always done it? I imagine you are very used to cases being ordered with the whole Master list though - I e including MT, MOVE, NMD etc. Do you know if this was some kind of special situation, or if something is up with the way you order in Orderportalen?

Since you still have OMIM-AUTO on case 22331, you could still add panels available for your institutes to the list that are always searchable. For that, go to the institute settings (e.g. from the caseS page, click Settings) and on the dropdown "Gene panels available for filtering" select the ones that you are interested in. After saving the settings, the selected panels will show up on your variantS pages.
<img width="630" alt="Screenshot 2022-10-24 at 10 57 54" src="https://user-images.githubusercontent.com/758570/197488493-8f24a38f-7b68-4c1e-b3ab-e40cfca4c904.png">

**Comment by dnil** [comment]
Or you just order the case again, with the panels you need. That way you also ensure that all variants in those genes are actually loaded. Since you have OMIM-AUTO on that case in particular, most should be there, but as you are aware, OMIM is not always up to date with the latest publications.

**Comment by dnil** [comment]
I'll close the issue with that, but feel free to reopen!

**Comment by NicoleCMMS** [comment]
Hej Daniel,
Unfortunately changing the list of gene panels within our institute settings does not give us all the information we had before the last Scout update.  In our gene panels we have manually currated the transcript connected with disease and what the inheritance pattern is. This information is vital for us to judge whether the variant is of interest or not.  If the gene panel is not run as default this information is not available to us.  Basically your suggested solution does not solve our problem.  All of the cases loaded over the weekend into Scout are now affected by this problem.

**Comment by dnil** [comment]
Ah, so you need it checked as a default panel. Sure, then you will have to order the panels you want as things are right now?

I take it you did check in Orderportalen if all looks well?  If so I'll double check the case on disk and see if I can see anything suspicious!

**Comment by dnil** [comment]
Hi, checked again, and everything on disk shows the example (fastshark) ordered with three panels: IEM, MT and OMIM-AUTO? Sounds odd with panels dissapearing though, so let me know when you have checked what you ordered, and we can transfer this issue over to the appropriate component program in the chain!

**Comment by NicoleCMMS** [comment]
Yes we ordered IEM and MOVE panels to be default but we have always had the entire list of clinical gene panels to choose from on the case page in order to add extra if necessary. If we change the default panel on the case page all information for that panel is available to us on the individual variant pages.  If we add another gene panel in the snv filter page we do not get this information. Basically we have to be able to add a gene panel as a default gene panel on the case page and for all cases loaded over the weekend this is not the case. If you want to see how it looked before the weekend check case number 22322 to see the difference.

**Comment by dnil** [comment]
I quite understand. From the perspective of Scout, nothing has happened over the weekend, and by the looks of it not for MIP either as the files on disk show the same story. 

Several of the Klinisk genetik cases do show the same pattern (selected panel, not master list) so I don't think it was any kind of manual issue when ordering. 

Whatever caused the difference likely happened somewhere in/between StarLIMS and in/between CG. Perhaps something with the master list file, with Orderportalen or the CG monolith. Let's see if we can't find someone who might know; @moahaegglund @moedarrah @annaengstrom?

**Comment by dnil** [comment]
Ah, I might have found the issueTransferring to CG! 

https://github.com/Clinical-Genomics/cg/pull/1585
